phalcon/application.zep

```diff
@@ -118,13 +118,31 @@ abstract class Application extends Injectable implements EventsAwareInterface
 	public function getModule(string! name) -> array | object
 	{
 		var module;
+		var exception;
 
 		if !fetch module, this->_modules[name] {
-			throw new Exception("Module '" . name . "' isn't registered in the application container");
+			let exception = new Exception("Module '" . name . "' isn't registered in the application container");
+  			this->_handleException(exception);
```

Comment:
Are you sure about this? I can't get the fire behaviour but I think that if no event is defined, the result is `NULL`. So `false` means other fire event status returned.

The `dispatcher`where I copied this, works that way, only throws exception if the result of calling `fire`is NOT false.